src/apiCalls.js

```diff
@@ -0,0 +1,49 @@
+export const getAllMovies = () => {
```

Comment:
Good work making a new file for the fetch requests and then exporting each individually to import into App. Also, good work on adding the conditional to the first .then to make sure our errors are being thrown correctly.